enforcer/src/main/java/org/wso2/micro/gateway/enforcer/server/EnforcerWorkerPool.java

```diff
@@ -0,0 +1,33 @@
+package org.wso2.micro.gateway.enforcer.server;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.LinkedBlockingQueue;
+import java.util.concurrent.ThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Class that holds the  thread pool that serves the external auth requests coming from the router component
+ */
+public class EnforcerWorkerPool {
+    private final BlockingQueue<Runnable> blockingQueue;
+    private final ThreadPoolExecutor executor;
+    private static final Logger log = LogManager.getLogger(EnforcerWorkerPool.class);
+
+    public EnforcerWorkerPool(int core, int max, int keepAlive, int queueLength, String threadGroupName,
+            String threadGroupId) {
+        if (log.isDebugEnabled()) {
+            log.debug("Using native util.concurrent package..");
```

Comment:
Is this log correct? didn't get the meaning?

```suggestion
            log.debug("Using native util.concurrent package.");
```